operations/operations/publishing/publishing-guidelines/documentation-style-guide.md

```diff
@@ -63,12 +63,32 @@ We use the Oxford \(serial\) comma in our documentation.
 
 ### Capitalization
 
+Capitalization is a work in progress and inconsistencies are actively being worked on. New documentation and UI/in-product content should follow this style guide.
+
+#### Documentation headings
+
+**H1**
+
 Use Title Case for document names/titles.
   - For example: "Deployment Guide for Organizations"
 
+**H2 onwards**
```

Comment:
Propose using `#####` to anchor to the specific section instead of using bold formatting